cmd/dataset_tokenizer/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/wbrown/gpt_bpe/cmd/dataset_tokenizer
 
-go 1.18
+go 1.22
```

Comment:
Have you confirmed that all the commands, tests and other affected components of gpt_bpe work during the shift from 1.18 to 1.22?

Reply:
All test cases passed. This was prompted by the debugger complaining about depreciation of `go 1.18`.